sys-apps/yarn/metadata.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+<use>
+	<flag name="system-node">Pull system-wide provided nodejs</flag>
```

Comment:
This package needs a maintainer. If you are willing to maintain this through the proxy maintainers project, please add &lt;maintainer&gt; blocks to the metadata as shown in the example in https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide#How_proxy_maintainership_works.